components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -57,6 +57,20 @@ var credentialHelper = &cobra.Command{
 		}()
 
 		repoURL, gitCommand := parseProcessTree()
+
+		// Starts another process which tracks the executed git event
+		gitCommandTracker := exec.Command("/proc/self/exe", "git-track-command", "--gitCommand", gitCommand)
+		err = gitCommandTracker.Start()
+		if err != nil {
```

Comment:
I would prefer that we don't fail credential helper if for some reason tracking does not work, just log as error and continue